### ISSUE TITLE
Badge toggling between Passing and Failing

### ISSUE THREAD

**Issue by ravichandranjv** [travis-ci/travis-ci#8125]
Moving between tabs causes the toggle between Passed and Failed status badges.

Associated with the behavior could be the **Inactive branches** (_with Jobs_) **that may have failed** and are being picked up on page refresh or moving between tabs. 

Maybe an option to clear **Inactive** branches and associated jobs could be useful?

**Comment by BanzaiMan** [comment]
Do you have a URL to show this problem?

**Comment by ravichandranjv** [comment]
This is a screenshot taken just now with the link. Can you access my Travis?

[https://travis-ci.org/ravichandranjv/TestOTP---Automated/builds](![image](https://user-images.githubusercontent.com/804860/28410335-f9b8ab32-6d5a-11e7-8f23-5e578b4f9d4b.png))

**Comment by BanzaiMan** [comment]
Yes, the URL is what I meant. I found it, though. So that's all good.

On https://travis-ci.org/ravichandranjv/TestOTP---Automated/builds, we are fetching a badge with wrong branch name somehow: https://travis-ci.org/ravichandranjv/TestOTP---Automated.svg?branch=add-license-1
